### PR TITLE
changes on designer and code

### PR DESCRIPTION
**Pull request by nyagishira**

### CODE REVIEW

**Comment by dannyparsons** [comment]
Have you tested if the dialog works? From my quick look I think it will give an error.

**Comment by nyagishira** [comment]
it is not changing a dataframe yet but the R code is running

**Comment by dannyparsons** [comment]
If it's not changes the labels of the factor then the code isn't working correctly. Have you tried getting the code from the log and running it in R to see what happens?

**Comment by nyagishira** [comment]
there are some changes that John did on the ucrFactor and i was hoping they
woulg be merged so that my dialog can run

File: instat/dlgLabels.vb [hunk 1]
```diff
@@ -46,4 +58,25 @@ Public Class dlgLabels
         TestOKEnabled()
         SetDefaults()
     End Sub
+
+    Private Sub ucrFactorLabels_CheckedChanged() Handles ucrFactorLabels.checkedChanged
+        ucrBase.clsRsyntax.AddParameter("new_levels", ucrFactorLabels.GetSelectedLevels())
```
Comment (dannyparsons) [inline instat/dlgLabels.vb]: Ask @Lunalo about the GetSelectedLevels() command. This is what he used, but it's not what you need. John, what does this return? Why is it not appropriate for what Mary wants here. What should the new_levels parameter be and is there another method in ucrFactor which could do it?

**Comment by nyagishira** [comment]
check this current code..it does not change the dataframe but R is running

**Comment by dannyparsons** [comment]
Can this be closed?

**Comment by Lunalo** [comment]
Hi @nyagishira , @dannyparsons  
Sorry for the delay, The way I understood the method above, it allows to concatenate a number of the the levels from the factor that we are working on.

**Pull request closed without merging**